### PR TITLE
Explain drainage_area when adding a resource

### PR DESCRIPTION
**Pull request by RobinIsTheBird**
_Note to the reviewer: the rest of the requirements in this issue are
wholly addressed in the addons repository, and are entirely independent
of the core work represented by this PR. I think you can review this PR
without reference to the coming addons PR._

When adding a resource, if the choice is RainGarden or Bioswale,
when it prompts for drainage_area, provide help text below the input.
Also, update the question text.

Changes:

1) `assets/css/sass/partials/pages/_addtree.scss`: style the help text.
2) `stormwater/*_add.html`: update the question text and add
   the explanation to the `create` tag for RainGarden and Bioswale,
   and direct interpolation of their terminology.
3) `treemap/field/div.html`: add conditional explanation markup.
4) `treemap/templatetags/form_extras.py`: add "`withhelp`" keyword to and
   content to the tag nodes, defaulting to `Field.help_text`, if any,
   and interpolate label and explanation with terminology.
5) `treemap/search_fields.py`: adapt to interface change stemming from
   changes to `form_extras`.
6) Bump the required rev on `django-debug-toolbar` in `dev-requirements`
   because the previous rev now crashes in our environment.

--

Connects to #2704
Depends on OpenTreeMap/otm-addons#1231

### CODE REVIEW

**Comment by RickMohr** [comment]
1. Enable Bioswales or Rain Gardens and **don't** change the terminology.
2. Add one. Step 4 ("About this Resource") is blank!

**Comment by RickMohr** [comment]
Otherwise it all looks great. Nice delving into the field tools. And DotDict.

**Comment by RickMohr** [comment]
image

To me it looks funny that the space between the field and the explanation is smaller than the space between the explanation lines.

**Comment by RobinIsTheBird** [comment]
Tnx, Rick. On it in the morning.

**Comment by RobinIsTheBird** [comment]
Temporarily closing the PR, since things that worked a minute ago stopped working.

**Comment by RobinIsTheBird** [comment]
Hadn't made all the changes I thought I had. Reopening now.

File: opentreemap/stormwater/templates/stormwater/Bioswale_add.html [hunk 1]
```diff
@@ -6,9 +6,17 @@
 
 {% block resource_add_details %}
 <div class="resource-question">
-  {# Translators: Bioswale.singular means Bioswale #}
-  {% trans "How large is the area that drains into the {Bioswale.singular}?"|interpolate_string:term as drainage_area %}
-  {% trans "Approximate size is fine. Do not include the area of the {Bioswale.singular} itself."|interpolate_string:term as explanation %}
+  {% captureas drainage_area %}
+    {% blocktrans with bioswale=term.Bioswale.singular %}
```
Comment (RickMohr) [inline opentreemap/stormwater/templates/stormwater/Bioswale_add.html]: As we discussed, I think we should stick with e.g. `Bioswale.terminology.singular` until we can make a wholesale switch to your proposed unification of singular/plural terms for both models and term substitutions.

**Comment by RickMohr** [comment]
As we discussed, I think we should stick with e.g. `Bioswale.terminology.singular` until we can make a wholesale switch to your proposed unification of singular/plural terms for both models and term substitutions.

**Comment by RickMohr** [comment]
I agree that your `captureas` template tag makes for more readable code than what you had before.

**Pull request merged**